static/src/stylesheets/module/facia/_items.scss

```diff
@@ -9,6 +9,7 @@
 @import 'item-layouts/fc-item--half';
 @import 'item-layouts/fc-item--three-quarters';
 @import 'item-layouts/fc-item--three-quarters-right';
+@import 'item-layouts/fc-item--three-quarters-tall';
```

Comment:
It didn't feel right to scope it to this container seeing as it was being added to a slice that can't be made exclusive for the `dynamic/story-package`. Also yeah I see no reason it shouldn't be part of our kit of parts for future use.